_topic_maps/_topic_map.yml

```diff
@@ -3106,7 +3106,7 @@ Topics:
     Topics:
     - Name: About the OADP 1.3 Data Mover
       File: about-oadp-1-3-data-mover
-    - Name: Backing up and restoring volumes by using CSI snapshots
+    - Name: Backing up and restoring volumes by using CSI snapshots data movement
```

Comment:
You will need to update the title in the assembly as well to reflect this change. The assembly title is "Backing up and restoring CSI snapshots" Not sure if that was intentional or not.
https://73847--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-backup-restore-csi-snapshots

Reply:
Thank you. Will do!